src/main/java/step3/model/Car.java

```diff
@@ -0,0 +1,24 @@
+package step3.model;
+
+public class Car{
+    private int position =0;
+    public  static final int MOVE_OR_STOP_VALUE = 4;
```

Comment:
intellij 의 자동정렬 기능을 사용해 보면 어떨까요?
설정에 따라 다를 수는 있지만
mac 의 경우 option + command + L 을 누르면 자동정렬이 된답니다!

```suggestion
    public static final int MOVE_OR_STOP_VALUE = 4;
```

Reply:
오 감사해요. 윈도우는 ctrl + alt + L 이네요👍